src/libopensc/cardctl.h

```diff
@@ -419,6 +428,15 @@ struct sc_cardctl_cardos_genkey_info {
 	unsigned short	fid;
 };
 
+struct sc_cardctl_cardos_pass_algo_flags {
+	unsigned int pass;
+	unsigned long card_flags; /* from card->flags i.e. user set */
+	unsigned long used_flags; /* as set by default */
+	unsigned long new_flags; /* set in pkcs15-cardos.c */
+	unsigned long ec_flags; /* for EC keys */
+	unsigned long ext_flags; /* for EC keys */
```

Comment:
clang-format suggested the following change:


```
-	unsigned long new_flags; /* set in pkcs15-cardos.c */
-	unsigned long ec_flags; /* for EC keys */
-	unsigned long ext_flags; /* for EC keys */
+	unsigned long new_flags;  /* set in pkcs15-cardos.c */
+	unsigned long ec_flags;   /* for EC keys */
+	unsigned long ext_flags;  /* for EC keys */
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/cardctl.h -->